manifests/plugin/cuda.pp

```diff
@@ -0,0 +1,78 @@
+# cuda plugin
```

Comment:
I would start directly with puppet-strings docs : https://github.com/puppetlabs/puppet-strings#documenting-puppet-classes-and-defined-types